src/components/IndexPage/HeroSection/HeroSection.view.tsx

```diff
@@ -0,0 +1,59 @@
+import {Container, Text, Button, Flex, Space} from '@mantine/core'
+import * as React from "react"
+import {IconBrandGithub, IconChevronsDown} from '@tabler/icons-react'
+import * as classes from './HeroSection.module.css'
+
+type Props = {
+    onExploreProjectsClick: () => void;
+}
+
+
+export default function HeroSectionView({onExploreProjectsClick}: Props): React.JSX.Element {
+    return <>
+        <Container size='xl' className={classes.inner}>
+            <h1 className={classes.title}>
+                Open-Source Software at{' '}
+                <Text component="span" variant="gradient" gradient={{from: 'blue', to: 'cyan'}} inherit
+                      className={classes.brand}>
+                    G-Research
+                </Text>
+            </h1>
+
+            <Text className={classes.description} c="dimmed">
+                We play an active role in the open-source community. We take responsibility for maintaining projects,
+                contributing to the continual development of essential software and supporting open-source foundations.
+                <Space component="br"/>
```

Comment:
Maybe two <br />'s here?  Somehow the single line break looks weird to me.

Reply:
hahah, of course github interpreted my "br"'s in that previous comment.  Genius.

Reply:
I added another br tag 👌